scripts/src/index.ts

```diff
@@ -0,0 +1,46 @@
+import * as http from "http";
+
+import { init as initConfig } from "./config";
+const config = initConfig("config/default.json");
```

Comment:
you're right. but this is special.
this `initConfig` must happen BEFORE other modules are imported, because those modules use `getConfig` and we need the config to be initialized before that happens.

Reply:
👍